### ISSUE TITLE
Potential issue with trajectory tracking 

### ISSUE THREAD

**Issue by BenCaunt** [First-Tech-Challenge-Team-20434-NCSSM/2022Offseason#2]
Line of issue: 
https://github.com/First-Tech-Challenge-Team-20434-NCSSM/2022Offseason/blob/c9ab6f08af1ba861f95f7ea67566974ec6c7cec4/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/Commands/DrivetrainCommands/DriveTrajectory.java#L37

There might be an issue here with using TurnOnlyControl as TurnOnlyControl regenerates the motion profile every time a new reference pose is sent.  This might still work just because each new angle will be relatively close to the previous however a potential concern I have is that the motion profile will prevent the robot from accurately following the trajectory.  

Regeneration of motion profile in TurnOnlyControl
https://github.com/First-Tech-Challenge-Team-20434-NCSSM/2022Offseason/blob/master/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Math/Controllers/TurnOnlyControl.java#L73

**Comment by jacobvm04** [comment]
That's true, perhaps just using an AngleController (with sqrt control as done in TurnOnlyControl) directly would yield better results?
